### PR TITLE
Enable dnf5 test

### PR DESCRIPTION
**Pull request by inknos**
This PR includes tests that *just work* with `@dnf5` tag.
It also includes functions to parse dnf4/5 stdout/err and exit codes. I also disables checking for transaction table stdout for dnf5

### CODE REVIEW

File: dnf-behave-tests/common/output.py [hunk 1]
```diff
@@ -124,6 +172,30 @@ def then_dnf_stdout_is(context, dnf_version):
         then_stdout_is(context)
 
 
+@behave.then("dnf4 stdout matches line by line")
```
Comment (kontura) [inline dnf-behave-tests/common/output.py:175]: I am not sure if you prefer this approach but just adding a note that all the new steps could use the `{dnf_version:dnf_version}` pattern just like `@behave.then("{dnf_version:dnf_version} stdout is")`.

Comment (inknos) [reply dnf-behave-tests/common/output.py:175]: didn't know, I definitely like it more. thanks

File: dnf-behave-tests/common/output.py [hunk 2]
```diff
@@ -54,6 +54,17 @@ def handle_reposync(expected, found, dnf5_mode):
     return expected, found
 
 
+@behave.then("{dnf_version:dnf_version} exit code is")
```
Comment (kontura) [inline dnf-behave-tests/common/output.py:57]: It seems the `{dnf_version:dnf_version}` pattern is being used before it is registered below (`behave.register_type(dnf_version=parse_dnf_version)`) so its not working.

File: dnf-behave-tests/common/output.py [hunk 3]
```diff
@@ -124,6 +135,41 @@ def then_dnf_stdout_is(context, dnf_version):
...
+    dnf5_mode = hasattr(context, "dnf") and context.dnf.dnf5_mode
+    if dnf_version == "dnf5" and dnf5_mode:
+        then_stderr_is(context)
+    if dnf_version == "dnf4" and not dnf5_mode:
+        then_stderr_is(context)
+
+
+@behave.then("{dnf_version:dnf_version} stdout matches line by line")
+def then_dnf_stdout_matches_line_by_line(context, dnf_version):
+    """
```
Comment (kontura) [inline dnf-behave-tests/common/output.py:153]: Empty doc descriptions are added on a couple places.

**Review by kontura — CHANGES_REQUESTED** [review]
Otherwise looks good.
All the marked tests are passing for me.

**Review by kontura — APPROVED** [review]

**Comment by kontura** [comment]
Great, thank you!

**Pull request merged**